Modules/TRD/src/DigitsTask.cxx

```diff
@@ -340,7 +362,7 @@ void DigitsTask::buildHistograms()
     getObjectsManager()->startPublishing(h);
   }
 
-  int cn = 0;
+  /*  int cn = 0;
```

Comment:
we usually advise not to comment out code but to remove it. Git will remember any ways.

Reply:
@Barthelemy it was more due to the fact that we have never replicated this problem. Invalid signature is using a 1 parameter update to a 2d spectra, which does not exist in the code. The segfault is clearly on these these std::array of spectra, but its done the same way as other places. So the underlying cause is a mystery. Its left in to be able to trivially put them back for testing.